src/screens/CreateCommunity/index.js

```diff
@@ -0,0 +1,327 @@
+        </TouchableWithoutFeedback>
+        <View style={styles.gap} />
+      </View>
+      <View style={styles.footer}>
+        <View style={styles.textSmallContainer}>
+          <Text style={styles.textSmall}>
+            You{"'"}ll appear as the first member of this community. You can switch your membership
+            to incognito at any time from the community page.
+          </Text>
+        </View>
+        <Button disabled={isNextButtonDisabled()} onPress={() => next()}>
+          Next
+        </Button>
+      </View>
+    </SafeAreaView>
+  );
+};
+
+export default CreateCommunity;
+
+const styles = StyleSheet.create({
+  containerInput: {
+    flexDirection: 'row',
+    marginTop: dimen.normalizeDimen(18),
+    marginBottom: dimen.normalizeDimen(20)
+  },
+  input: {
+    borderWidth: dimen.normalizeDimen(1),
+    borderRadius: dimen.normalizeDimen(8),
+    borderColor: COLORS.gray210,
+    backgroundColor: COLORS.gray110,
+    color: COLORS.white,
+    paddingHorizontal: dimen.normalizeDimen(23),
+    paddingVertical: dimen.normalizeDimen(13),
+    width: '100%'
+  },
+  container: {
+    flex: 1,
+    backgroundColor: COLORS.almostBlack
+  },
+  btnNext: {marginTop: dimen.normalizeDimen(16)},
+  gap: {flex: 1},
+  icon: {
+    width: dimen.normalizeDimen(14),
+    height: dimen.normalizeDimen(14),
+    position: 'absolute',
+    bottom: dimen.normalizeDimen(-5),
+    left: dimen.normalizeDimen(19)
+  },
+  image: {
+    height: dimen.normalizeDimen(52),
+    width: dimen.normalizeDimen(52),
+    borderRadius: dimen.normalizeDimen(26)
+  },
+  title: {
+    fontFamily: fonts.inter[600],
+    fontSize: normalizeFontSize(16),
+    color: COLORS.white,
+    marginTop: dimen.normalizeDimen(24)
+  },
+  desc: {
+    fontFamily: fonts.inter[400],
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(24),
+    color: COLORS.gray510,
+    marginTop: dimen.normalizeDimen(6)
+  },
+  content: {
+    paddingTop: dimen.normalizeDimen(20),
+    paddingHorizontal: dimen.normalizeDimen(20)
+  },
+  keyboardavoidingview: {
+    flex: 1,
+    justifyContent: 'flex-end'
+  },
+  textMessage: (color) => ({
+    fontSize: normalizeFontSize(12),
+    color,
+    fontFamily: fonts.inter[400],
+    marginTop: dimen.normalizeDimen(6)
+  }),
+  parentIcon: {
+    width: '10%',
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  parentInfo: {
+    width: '90%'
+  },
+  footer: {
+    position: 'absolute',
+    bottom: 0,
+    height: dimen.normalizeDimen(112),
+    width: '100%',
+    paddingHorizontal: dimen.normalizeDimen(20),
+    paddingBottom: dimen.normalizeDimen(20),
+    backgroundColor: COLORS.almostBlack,
+    flexDirection: 'column',
+    justifyContent: 'space-between'
+  },
+  textSmallContainer: {
+    flex: 1,
+    justifyContent: 'center'
+  },
+  textSmall: {
+    fontSize: normalizeFontSize(10),
+    fontFamily: fonts.inter[400],
+    textAlign: 'center',
+    color: COLORS.gray500
+  }
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `return` statement in the component is quite large and could be broken down into smaller components for better readability and maintainability. For example, the `Header` and `Input` could be separate components. However, this is a larger refactor and may not be necessary depending on the overall structure of your application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->